eoscostarica/proton-affiliate
# docs(contracts): update ricardian clauses

### Update ricaridan clauses

### What does this PR do?

- Remove old clauses and add new ones

### Steps to test

1. Check content is right
2. Check only Ricardian files are changed and abi
